Packages/vcs/Lib/vcsvtk/boxfillpipeline.py

```diff
@@ -0,0 +1,383 @@
+from .pipeline2d import Pipeline2D
+from .. import vcs2vtk
+
+import numpy
+import vcs
+import vtk
+import warnings
+
+class BoxfillPipeline(Pipeline2D):
+    """Implementation of the Pipeline interface for VCS boxfill plots.
+
+    Internal variables:
+        - self._legend: Contour labels.
```

Comment:
legend actually has another meaning in VCS vs "labels" the legend is used to display which lines represent what. If think "labels" or "ContourLabels" might be better here.